spring-webmvc/src/main/java/org/springframework/web/servlet/mvc/method/annotation/ReactiveTypeHandler.java

```diff
@@ -515,11 +516,11 @@ private static class ContextSnapshotHelper {
 
 		public static Object writeReactorContext(Object returnValue) {
 			if (Mono.class.isAssignableFrom(returnValue.getClass())) {
-				ContextSnapshot snapshot = ContextSnapshot.captureAll();
+				ContextSnapshot snapshot = ContextSnapshotFactory.builder().build().captureAll();
```

Comment:
I think there is no real reason to require 1.0.3 here. Couldn't we continue using the deprecated method a while longer and suppress the deprecation?

Reply:
The build fails if I don't do this :shrug:

Reply:
Yes, exactly. The current used method is marked as deprecated

Reply:
Ah ok, let me fix that then